modules/openshift-developer-cli-pushing-the-init-image-directly-on-windows.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * openshift_cli/openshift_developer_cli/using_odo_in_a_restricted_environment/pushing-the-odo-init-image-to-an-internal-registry-directly.adoc
+
+[id="pushing-the-init-image-directly-on-windows.adoc_{context}"]
+
+= Pushing the `init` image directly on Windows
+
+.Procedure
+
+. Enable the default route:
++
+----
+PS C:\> oc patch configs.imageregistry.operator.openshift.io cluster -p '{"spec":{"defaultRoute":true}}' --type='merge' -n openshift-image-registry
+----
+
+. Get a wildcard route CA:
++
+----
+PS C:\> oc get secret router-certs-default -n openshift-ingress -o yaml
+apiVersion: v1
+data:
+  tls.crt: **************************
+  tls.key: ##################
+kind: Secret
+metadata:
+  [...]
+type: kubernetes.io/tls
+----
+
+. Use `base64` to encode your internal registry certification authority (CA) content:
++
+----
+PS C:\> echo <tls.crt> | base64 -d > ca.crt
+----
+
+. Trust a CA in your client platform/ Execute the following command in PowerShell as administrator:
```

Comment:
As an administrator, trust a CA in your client platform by executing the following command in PowerShell: